components/concept/sync/src/main/java/mozilla/components/concept/sync/OAuthAccount.kt

```diff
@@ -293,6 +293,8 @@ sealed class AuthType {
 
 /**
  * Observer interface which lets its users monitor account state changes and major events.
+ * (XXX - there's some tension between this and the
+ * mozilla.components.concept.sync.AccountEvent we should resolve!)
```

Comment:
Can you file & link an issue for this? I don't think we need to block this PR on updating this stuff.